utils/common/src/test/kotlin/UtilsTest.kt

```diff
@@ -65,28 +64,4 @@ class UtilsTest : WordSpec({
             )
         }
     }
-
-    "isMavenCentralUrl" should {
```

Comment:
Would be nice to move these into `PackageProviderTest` class.

Reply:
> Would be nice to move these into `PackageProviderTest` class.

There's no such test class yet (remember our [discussion of the temporary test that I added during development](https://github.com/oss-review-toolkit/ort/pull/6472#discussion_r1103580937)), and as such a test for `PackageProvider` would need to cover way more URLs, I'd prefer to address that in a separate PR.

Reply:
I do remember that one, I brought it up again because now it's a different context, which is dropping test coverage.

I don't see what the problem would be with creating a new class and moving the function there, even if the class does not cover all cases but only the ones from the other test. I'll leave it optional anyway.

Reply:
> I don't see what the problem would be with creating a new class

Just time constraints. But I've now added a start at https://github.com/oss-review-toolkit/ort/pull/6499.